### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by russgold**
@tbarnes-us Made some suggestions for improving the main page docs, and I have tried to incorporate them, here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +50,7 @@ Note that there are two parts to the configuration. The optional top portion def
 | `query_sync.interval` | The interval, in seconds, at which the service will be queried. Defaults to 10. |
 | `metricsNameSnakeCase` | If true, metrics names will be converted to snake case. Defaults to false. |
 | `domainQualifier` | If true, the domain name will be included as a qualifier for all metrics. Defaults to false. |
-| `restPort` | Optional. Overrides the port on which the exporter should contact the REST API. Needed if the exporter cannot find the REST API. |
+| `restPort` | Optional, used in the web application only. Overrides the port on which the exporter should contact the REST API. Needed if the exporter cannot find the REST API. The most common case is running on a system with the admin port enabled. In that case, you must specify the admin port in this field and access the exporter via the SSL port. |
 
 The `query` field is more complex. Each query consists of a hierarchy of the [MBeans](https://docs.oracle.com/middleware/1221/wls/WLMBR/core/index.html), starting relative to `ServerRuntimes`.
```
Comment (rosemarymarano) [inline README.md:55]: Again, if you want a later version of the MBean Reference (12.2.1.3.0), https://docs.oracle.com/middleware/12213/wls/WLMBR/core/index.html

**Review by rosemarymarano — COMMENTED** [review]
A few minor edits

File: README.md [hunk 2]
```diff
@@ -3,9 +3,18 @@ WebLogic Monitoring Exporter
...
-and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains via the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - a [web application](#web-application) that you deploy to the managed server from which metrics are to be extracted. 
+ You may include a configuration file directly in the WAR, and you may temporarily modify the configuration in a 
+ running system via a web form.
```
Comment (tbarnes-us) [inline README.md:14]: add something like "you can configure the web-app to access its configuration file from an external central configuration coordinator"

File: README.md [hunk 3]
```diff
@@ -3,9 +3,18 @@ WebLogic Monitoring Exporter
 
 [![Build Status](https://travis-ci.org/oracle/weblogic-monitoring-exporter.svg?branch=master)](https://travis-ci.org/oracle/weblogic-monitoring-exporter)
 
-The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
-and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains via the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later.
```
[the remark this answers is not in the document]
Comment (russgold) [reply README.md:8]: Thanks. Do you recommend using that one by preference?

File: README.md [hunk 4]
```diff
@@ -3,9 +3,18 @@ WebLogic Monitoring Exporter
...
-The WebLogic Monitoring Exporter uses the [WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111) to scrape runtime information and then exports [Prometheus](http://prometheus.io)-compatible metrics.
-It is available both as a [web application](#web-application) to be deployed to the WebLogic Server (WLS) instance, version 12.2.1 or later, from which you want to get metrics, 
-and also as a [sidecar](#sidecar), which runs as its own process, and which is configured to contact a WLS instance.
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains via the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - a [web application](#web-application) that you deploy to the managed server from which metrics are to be extracted.
```
Comment (tbarnes-us) [inline README.md:12]: i assume one might want admin server metrics, right?

if so, "managed server" --> "WebLogic server"

Comment (rosemarymarano) [reply README.md:12]: "managed server" --> "WebLogic Server"

File: README.md [hunk 5]
```diff
@@ -41,7 +50,7 @@ Note that there are two parts to the configuration. The optional top portion def
 | `query_sync.interval` | The interval, in seconds, at which the service will be queried. Defaults to 10. |
```
Comment (tbarnes-us) [inline README.md:50]: Please make it clear that the query_sync settings are all 'web-app-only'

File: README.md (hunk 4 above)
Comment (russgold) [reply README.md:12]: No, this is only for managed servers.

File: README.md [hunk 6]
```diff
@@ -175,7 +184,7 @@ Use https | `false` | `WLS_SECURE`
 
 ## Configure the exporter
 
-The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
+You configure the sidecar by sending a PUT request to the path `/configuration`. 
 You can do this with `curl`; when used with the [operator](https://github.com/oracle/weblogic-kubernetes-operator/),
```
Comment (tbarnes-us) [inline README.md:189]: The wording kind of implies the operator is using some sort of secret F&F API.  How about wording something like:

A sidecar is configured by sending an HTTP PUT request to the path `/configuration`; you can do this with `curl`.  

When a sidecar is used with WebLogic domains that are deployed using the [WebLogic Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/), the WKO domain resource can be configured to automatically deploy and send configuration the sidecar.

File: README.md [hunk 7]
```diff
@@ -3,9 +3,18 @@ WebLogic Monitoring Exporter
...
+[WLS RESTful Management API](https://docs.oracle.com/middleware/1221/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - a [web application](#web-application) that you deploy to the managed server from which metrics are to be extracted. 
+ You may include a configuration file directly in the WAR, and you may temporarily modify the configuration in a 
+ running system via a web form.
+ 
+ - a [separate process](#sidecar) that is run alongside a managed server. You supply the configuration to such a
+process with a PUT command, as described below.
```
Comment (tbarnes-us) [inline README.md:18]: This bullet should also mention the ability to have WKO deploy and configure the side-car....

File: README.md [hunk 8]
```diff
@@ -3,9 +3,19 @@ WebLogic Monitoring Exporter
...
+The WebLogic Monitoring Exporter is a [Prometheus](http://prometheus.io)-compatible exporter of metrics from 
+WebLogic Server (WLS) instances, which it obtains by using the 
+[WLS RESTful Management API](https://docs.oracle.com/middleware/12213/wls/WLRUR/overview.htm#WLRUR111), available in version 12.2.1 or later. 
+Metrics are selected using a [YAML configuration file](#Configuration).
+
+The exporter is available in two forms:
+ - A [web application](#web-application) that you deploy to the Managed Server from which metrics are to be extracted. 
+ You may include a configuration file directly in the WAR file, and you may temporarily modify the configuration in a 
+ running system by using a web form. If a [coordination configurator](config_coordinator/README.md) is running and configured, 
+ that temporary configuration will be sent to all servers configured to use it.
```
Comment (tbarnes-us) [inline README.md:15]: Would the following be more precise?

You may embed a configuration file directly in the web application's WAR file or configure the web-application to obtain its configuration from a central [coordination configurator](config_coordinator/README.md).  In addition, you can temporarily modify the configuration dynamically by using a web form in your browser, and if a coordination configurator is setup, then any web-form submissions will be propagated to all servers that use the coordinator.

File: README.md [hunk 9]
```diff
@@ -175,9 +185,8 @@ Use https | `false` | `WLS_SECURE`
 
 ## Configure the exporter
 
-The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
-You can do this with `curl`; when used with the [operator](https://github.com/oracle/weblogic-kubernetes-operator/), 
-the operator will automatically configure the exporter. 
+You configure the sidecar by sending a PUT request to the path `/configuration`. 
+You can do this with `curl`; that is how the operator does it.
```
Comment (tbarnes-us) [inline README.md:189]: operator -> [WebLogic Server Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/)

File: README.md [hunk 10]
```diff
@@ -147,8 +158,7 @@ the version number to simplify deployment to WebLogic Server.
 
 # Sidecar
 
-The sidecar is a standalone process that runs the exporter. It is primarily intended for use by the
-[WebLogic Server Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/).
+The sidecar is a standalone process that runs the exporter.
```
Comment (tbarnes-us) [inline README.md:161]: It is an alternative to using the web-app.

File: README.md [hunk 11]
```diff
@@ -147,8 +158,7 @@ the version number to simplify deployment to WebLogic Server.
 
 # Sidecar
 
-The sidecar is a standalone process that runs the exporter. It is primarily intended for use by the
```
Comment (tbarnes-us) [inline README.md:150]: The 'web-app' section above fails to mention the coordination configurator, and is imprecise about the usage of the web-form. It should be updated to match the contents of the introduction.

File: README.md (hunk 4 above)
Comment (tbarnes-us) [reply README.md]: There is an ongoing discussion about this in slack.

**Review by rosemarymarano — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (russgold) [reply README.md]: I have change it to say server instance. admin/managed/independent are not relevant.

File: README.md (hunk 8 above)
Comment (russgold) [reply README.md:15]: No, that's not actually correct at all.

File: README.md (hunk 9 above)
Comment (russgold) [reply README.md:189]: I've already called it that once in the doc. @rosemarymarano do I need to use the full name each time?

File: README.md (hunk 11 above)
Comment (russgold) [reply README.md:150]: It's in the intro:
```
You may include a configuration file directly in the WAR file, and you may temporarily modify the configuration in a 
 running system by using a web form. If a [coordination configurator](config_coordinator/README.md) is running and configured, 
 that temporary configuration will be sent to all servers configured to use it.
 ```

File: README.md (hunk 10 above)
Comment (russgold) [reply README.md:161]: Strictly speaking, they are alternatives to each other; I would think that was obvious by the fact that both are listed as options.

File: README.md (hunk 9 above)
Comment (rosemarymarano) [reply README.md:189]: Nope; it's ok as "the operator."

File: README.md (hunk 10 above)
Comment (tbarnes-us) [reply README.md:161]: These are widely separated parts of the documentation - si not so obvious.

File: README.md (hunk 8 above)
Comment (tbarnes-us) [reply README.md:15]: Then can you make the original wording more helpful? It's not clear to me (at least) how configuration initialization and configuration changes are submitted to the coordinator.  This is nice information to have in an introduction.

File: README.md (hunk 9 above)
Comment (tbarnes-us) [reply README.md:189]: Not to quibble, but this is a relatively big piece of doc,  there are only two mentions of the operator, and the original usage and second usage are widely separated, it doesn't hurt to repeat in that situation (thus my comment).

File: README.md (hunk 11 above)
Comment (tbarnes-us) [reply README.md:150]: The 'web-app' section is a reference for the 'web-app', it should be complete.  In general, documentation about a specific feature should be relatively complete in of itself, even if a part of the feature is described in an intro/summy.

**Pull request merged**